Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_False_Positive_Incident_Handling_README.md

```diff
@@ -5,35 +5,35 @@ This playbook handles false-positive incident closures for Cortex XDR - Malware
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
-Cortex XDR - Unisolate Endpoint
+* Cortex XDR - Unisolate Endpoint
 
 ### Integrations
-CortexXDRIR
+* CortexXDRIR
 
 ### Scripts
 This playbook does not use any scripts.
 
 ### Commands
-* xdr-allowlist-files
-* setIndicators
 * closeInvestigation
+* setIndicators
+* xdr-allowlist-files
 
 ## Playbook Inputs
 ---
 
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| Comment | Add a comment to close this incident. | XSOAR Incident #${incident.id} | Optional |
-| Reason | Possible values:<br/> * Unknown <br/>* TruePositive<br/>* FalsePositive | FalsePositive | Optional |
+| Comment | Add comment to close this incident. | XSOAR Incident #${incident.id} | Optional |
+| Reason | Choose From - "Unknown" / "TruePositive" / "FalsePositive" | FalsePositive | Optional |
 | AllowTag | The approving tag name for found indicators. | AllowTag | Optional |
-| AutoUnisolation | Whether automatic un-isolation is allowed. | False | Optional |
```

Comment:
So now we are using this input?

Reply:
yes it used in the "Auto-Unisolate" task